hack/pull_kfserving_upstream.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Usage:
+# This script should be run from kubeflow/manifest repo's root:
+# ./hack/pull_kfserving_upstream.sh
+
+set -ex
+
+# Please edit the following version before running the script to pull new
+# kfserving version.
+export KFSERVING_VERSION=v0.4.0
+export KFSERVING_SRC_REPO=https://github.com/kubeflow/kfserving.git
+
+if [ -d kfserving/upstream ]; then
+    rm -rf kfserving/upstream
+fi
+kpt pkg get $KFSERVING_SRC_REPO/install/$KFSERVING_VERSION kfserving/upstream
```

Comment:
Just for my knowledge, what does `kpt pkg` command do ?

Reply:
Pretty much just syncs what is in the targeted kfserving repo directory with the local manifests directory. In this case, it just syncs whatever is in `kfserving/install/v0.4.0`. I am not particularly tied to this method as I was namely just following what was being [done for pipelines](https://github.com/kubeflow/manifests/blob/master/hack/pull_kfp_upstream.sh). The idea here is that you just increment the value in the script then run it when a new version of kfserving is out, so it's more of a convenience  method that can help get the latest kfserving yaml file. Ideally whatever is in the `kfserving/upstream` folder in this repo matches closely to what you have in the kfserving repo.